crates/rover-client/src/query/supergraph/fetch.graphql

```diff
@@ -0,0 +1,17 @@
+query FetchSupergraphQuery($variant: String!, $graphId: ID!) {
+  frontendUrlRoot
+  service(id: $graphId) {
+    schema(tag: $variant) {
+      hash
+    }
+    mostRecentCompositionPublish(graphVariant: $variant) {
```

Comment:
This will get the latest composition rather than the core schema currently running in gateways.

If you want the core schema running on gateways
`service.schemaTag.compositionResult.supergraphSdl`
Note this will never have errors because errors mean no core schema is generated so we can't send it to gateways.

Reply:
aahhh!! yes this is exactly what I'm looking for, thanks @jsegaran